internal/controllers/users.go

```diff
@@ -156,12 +156,17 @@ func (s Server) AddUsages(ctx echo.Context) error {
 	if resourceName == "" {
 		return ctx.JSON(http.StatusBadRequest, model.ErrorResponse("Invalid Resource Name", http.StatusBadRequest))
 	}
+	usageValueString := ctx.Param("usage_value")
+	if usageValueString == "" {
+		return ctx.JSON(http.StatusBadRequest, model.ErrorResponse("Invalid Usage Value", http.StatusBadRequest))
+	}
+	UsageValueFloat := ParseFloat(usageValueString)
```

Comment:
There should probably be some more error checking in case the parameter is formatted incorrectly.